/**: 
  ros__parameters:
    env_vg_topic: "env_builder_node/environment_voxel_grid" # topic on which env_builder is publishing the environment 
    id: 0 # id of the agent; needs to be modified in the launch file in case we have multiple agents
    voxel_grid_range: [20.0, 20.0, 6.0] # voxel grid size centered around the agent position
    world_frame: "world" # frame for publishing the voxel grid in
    free_grid: True # if true set all non occupied voxels to free
    inflation_dist: 0.3 # all voxels within that distance to an occupied voxel are set as occupied
    potential_dist: 1.5 # all voxels within that distance to an occupied voxel are set as a potential field
    potential_pow: 4.0 # power of the potential field distance formula for setting the voxel values in the potential field (1-d/potential_dist)^potential_pow_ * h_max
    fov_x: 1.57 # camera width fov in radians
    fov_y: 1.57 # camera length fov in radians
    fov_y_offset: 0.0 # camera tilt up in radians 
    frustum_length: 2.0 # frustum length for visualiztion in rviz
    limited_fov: false # if true use limited_fov, if false use 360 degree vision
